.scalafmt.conf

```diff
@@ -0,0 +1 @@
+version = "3.0.5"
```

Comment:
not sure how you want to use this new RPS, but shouldn't we limit each branch to contain only the changes required for the training?